java101/src/com/java101/HeroService.java

```diff
@@ -1,52 +1,46 @@
 package com.java101;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;;
+
 public class HeroService {
 
-	public static void main(String[] params) {
+	public void createHero() throws ParseException, IOException {
```

Comment:
Nazwa tej metody już tu nie pasuje, nie? :) Wymyśl jakąś inną która opisuje co to robi